_includes/layouts/post.njk

```diff
@@ -1,8 +1,9 @@
 ---
 layout: layouts/base.njk
 ---
+{# TODO: add a11y-light.css and adjust for light & dark themes #}
```

Comment:
Would be nice to load either the dark or light syntax highlighting theme based on `prefers-color-scheme` and the theme override, but inlining the CSS in prod makes this tricky.